documents/knowledge/elements.yaml

```diff
@@ -765,7 +765,7 @@
   tests:
     - "[20] : [1,2,4,5,10,20]"
     - "[1] : [1]"
-    - '["adbcdbcd"] : {"b","c","d","bc","cd","bcd","db","dbc"}'
+    - '["adbcdbcd"] : ["d", "db", "dbc", "b", "bc", "bcd", "c", "cd"]'
```

Comment:
The only problem with this is that if you make `K` prefixes for strings, `¦` becomes useless on strings (because it's cumulative sums). But then if you make `K` contiguous sublists for lists, there's no one-byte way to get list prefixes.